### ISSUE TITLE
Updating org-journal breaks capture

### ISSUE THREAD

**Issue by omidmnz** [bastibe/org-journal#218]
Using capture, among some other functionality in Org mode results in the following error after updating org-journal to the [latest](https://github.com/bastibe/org-journal/commit/e11b5270ac95e42e5927a1d5a05f105403fc7a79) in master branch:
```
Capture abort: Symbol's function definition is void: org-journal-update-auto-mode-alist
```
Reverting org-journal to my [previously installed version](https://github.com/bastibe/org-journal/commit/664c08e12cde19ce7dca645ba9accecda7266c32) (or uninstalling it) solves the issue.
I am using Org mode version 9.3.6 (release_9.3.6-449-gb99357) on Emacs 26.3.

**Comment by cslux** [comment]
Should be fixed via  3b7f7f6.

Sorry for the inconvenience.

**Comment by omidmnz** [comment]
Can confirm.
Thanks for the quick fix.
